### ISSUE TITLE
Device Signal Level/Noise/SNR/Quality Panel - Units Issue

### ISSUE THREAD

**Issue by wifinigel** [cucumber-tony/cucumber-frontend#95]
The panel that displays the signal level, SNR, Noise and Quality appears to have an issue with the units displayed for each metric:

SNR values are dispalyed in dBm & should be dB

![image](https://cloud.githubusercontent.com/assets/19818404/16236985/4df2303e-37d2-11e6-889a-a38ee1352f0f.png)

Signal level values are displayed in dB and should be dBm:

![image](https://cloud.githubusercontent.com/assets/19818404/16237007/61538740-37d2-11e6-9ec2-cc5ed94482b2.png)

Noise levels ate shown in dB and should be dBm:

![image](https://cloud.githubusercontent.com/assets/19818404/16237018/6f76c9b8-37d2-11e6-89b7-3dfe62aa4286.png)

The units for quality are not clear - should these be % figures?

![image](https://cloud.githubusercontent.com/assets/19818404/16237037/825df696-37d2-11e6-8d0c-301105aafec1.png)

Also, the control highlighted below does not seem to do anything when toggled?

![image](https://cloud.githubusercontent.com/assets/19818404/16237058/97c8d654-37d2-11e6-86ba-bb01182afbd1.png)

Also, could the WLAN names wlan0 & wlan1 be displayed as the actual SSID that they correspond to?

**Comment by simonmorley** [comment]
Got this here: https://github.com/cucumber-tony/cucumber-frontend/issues/16

Will close in favour of that, despite pwetty graphs

**Comment by simonmorley** [comment]
Ignore me, I'd mis-read.

**Comment by simonmorley** [comment]
The figures have been sorted, we were looping through in wrong order.

@AndrewMargarit and I discussed the quality and believe it doesn't really have a suffix. If it's confusing or misleading, we can add the percentage. I've added short-term - you two should thrash it out.

Right now, displaying the SSID isn't possible. I've marked as an enhancement.  And that button has not been implemented yet.

-- Enhancements --

Add SSID instead of interface name.
